recipes/autokeras/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "autokeras" %}
+{% set version = "1.0.20" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/autokeras-{{ version }}.tar.gz
+  sha256: 3d0f09629abb9b98f9059e3c8b9d39b4acd7f6dcde6d0584146a02035159141a
+
+build:
+  number: 0
+  skip: True  # [win or py<37]
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"  # [not win]
```

Comment:
Unfortunately, this is not possible due to https://github.com/conda-forge/tensorflow-feedstock/issues/124
**Windows-Builds** will crash... sorry I made a force push to overwrite the failed **Windows-Builds**

Reply:
Right, we can ignore the failed Windows builds this results in because it would be a noarch packages. noarch packages are packages that will work on every platform, but (by default) are only built on Linux. So the build would succeed. The package wouldn't be *installable* on Windows until we get new builds for tensorflow on Windows.

The benefit of noarch is it drastically cuts down on the number of builds (makes it easier when new versions of Python appear).

Reply:
Thx, @dopplershift , for clarifying this. I thought a failed build would not be legal.